src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -113,41 +133,119 @@ function WorkspaceCategoriesPage({policy, policyCategories, route}: WorkspaceCat
         </View>
     );
 
-    const navigateToCategoriesSettings = () => {
-        Navigation.navigate(ROUTES.WORKSPACE_CATEGORIES_SETTINGS.getRoute(route.params.policyID));
+    const navigateToCategorySettings = (category: PolicyOption) => {
+        Navigation.navigate(ROUTES.WORKSPACE_CATEGORY_SETTINGS.getRoute(route.params.policyID, category.text));
     };
 
-    const navigateToCategorySettings = (category: PolicyForList) => {
-        Navigation.navigate(ROUTES.WORKSPACE_CATEGORY_SETTINGS.getRoute(route.params.policyID, category.text));
+    const navigateToCategoriesSettings = () => {
+        Navigation.navigate(ROUTES.WORKSPACE_CATEGORIES_SETTINGS.getRoute(route.params.policyID));
     };
 
     const navigateToCreateCategoryPage = () => {
         Navigation.navigate(ROUTES.WORKSPACE_CATEGORY_CREATE.getRoute(route.params.policyID));
     };
 
-    const isLoading = !isOffline && policyCategories === undefined;
+    const dismissError = (item: PolicyOption) => {
+        Policy.clearCategoryErrors(route.params.policyID, item.keyForList);
+    };
+
+    const selectedCategoriesArray = Object.keys(selectedCategories).filter((key) => selectedCategories[key]);
+
+    const getHeaderButtons = () => {
+        if (selectedCategoriesArray.length > 0) {
+            const isAllEnabled = selectedCategoriesArray.every((categoryName) => policyCategories?.[categoryName].enabled);
 
-    const headerButtons = (
-        <View style={[styles.w100, styles.flexRow, isSmallScreenWidth && styles.mb3]}>
-            {!PolicyUtils.hasAccountingConnections(policy) && (
+            const options: Array<DropdownOption<DeepValueOf<typeof CONST.POLICY.CATEGORIES_BULK_ACTION_TYPES>>> = [
+                {
+                    icon: Expensicons.Trashcan,
+                    text: translate('workspace.categories.deleteCategories'),
+                    value: CONST.POLICY.CATEGORIES_BULK_ACTION_TYPES.DELETE,
+                    onSelected: () => {
+                        setSelectedCategories({});
+                        deleteWorkspaceCategories(route.params.policyID, selectedCategoriesArray);
+                    },
+                },
+            ];
+
+            if (isAllEnabled) {
```

Comment:
I don't think this is correct. We want to include this option is some categories are enabled, not all